app/controllers/users_controller.rb

```diff
@@ -38,7 +38,9 @@ def index
       users = users.where(:status => @params[:status]) if @params[:status]
       users = users.where(:creation_ip => @params[:ip]) if @params[:ip]
 
-      @users_count = users.count
+      @users_count = users.limit(501).count
+      @users_count = "500+" if @users_count > 500
```

Comment:
I was told [this](https://github.com/openstreetmap/openstreetmap-website/pull/4201#discussion_r1310176384) about "number+".

Reply:
Thanks - I've fixed that now.